### ISSUE TITLE
Makefile message is not consistent with the code or documentation

### ISSUE THREAD

**Issue by ilitzroth** [haskell/haskell-mode#720]
From the makefile.

```
check-emacs-version :
	@$(BATCH) --eval "(when (< emacs-major-version 24)					\
                            (message \"Error: haskell-mode requires Emacs 23 or later\")	\
```

**Comment by gracjan** [comment]
Will you create a pull request?

Note: we do support Emacs 23 so it should be `(< emacs-major-version 23)`.

**Comment by ilitzroth** [comment]
ok... I'll get do that this evening.
Greetz,
i

**Comment by gracjan** [comment]
Fixed by  #722.
